src/state/actions/__test__/interests.test.js

```diff
@@ -0,0 +1,104 @@
+import React from 'react';
+import * as rtl from '@testing-library/react';
+import configureStore from 'redux-mock-store';
+import axios from 'axios';
+import MockAdapter from 'axios-mock-adapter';
+import thunk from 'redux-thunk';
+import * as types from '../../types/index';
+import { getUsersInterests, getInterests } from '../interests';
+
+const middlewares = [thunk];
+const mockStore = configureStore(middlewares);
+const mock = new MockAdapter(axios);
+
+const interests = {
+  id: 1,
+  interest: 'AI Engineer',
+};
```

Comment:
The `interests` endpoint is supposed to return an array of objects and not an `object`.